t/cobrand/tfl.t

```diff
@@ -296,6 +298,34 @@ subtest "test report creation anonymously by button" => sub {
     $mech->not_logged_in_ok;
 };
 
+subtest "test users with tfl email not asked to update their FMS password" => sub {
+    $tfl_staff->set_extra_metadata('last_password_change', time() - 200000);
+    $mech->get_ok('/auth');
+    FixMyStreet->test_mode(0);
+    $mech->submit_form_ok(
+        { with_fields => { username => $tfl_staff->email, password_sign_in => 'password' } },
+        "sign in using form"
+    );
+    FixMyStreet->test_mode(1);
+    $tfl_staff->set_extra_metadata('last_password_change', time());
+    is $mech->uri->path, '/my', "logged in to My Account page";
+    $mech->content_lacks('Your password has expired, please create a new one below');
+};
+
+subtest "test users without tfl password asked to update their FMS password" => sub {
+    $tfl_staff->set_extra_metadata('last_password_change', time() - 200000);
```

Comment:
You're not using tfl_staff here so no need to change and reset its last_password_change in this test.